build_web_compilers/pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: build_web_compilers
-version: 3.0.0
+version: 3.1.0-dev
```

Comment:
If you are ready to release this we can go ahead and do that and you can drop the `-dev` (from the changelog also).

Reply:
Not ready yet as we need the SDK to actually produce symbols first. Will update when ready.

Reply:
Ok, are you fine with waiting to land this until then? I don't want to risk an accidental release of this in a state where it doesn't have the proper constraints.

Reply:
Definitely!